package.json

```diff
@@ -38,7 +38,8 @@
   },
   "scripts": {
     "build": "tree-sitter generate && node-gyp build",
-    "test": "tree-sitter test && ./script/parse-examples.rb",
```

Comment:
@aymannadeem is this OK to switch over to the bash script we use in other `tree-sitter-<lang>` projects? No worries if not as it would be easy to run both here, but with a quick review they appear to be doing similar things.

Reply:
@tclem I think that @aymannadeem and I wrote that script in ruby just because we hadn't figured out the `xargs -n` trick that you introduced, so we were having to implement the batching manually. I would say go ahead and replace it with the bash script.

Reply:
yeah, originally had this (https://github.com/tree-sitter/tree-sitter-java/issues/25) but it was taking too long and we just thought ruby would be easier to debug